License

```diff
@@ -1,6 +1,8 @@
-SQLite Foreign Data Wrapper for PostgreSQL
+SQLite Foreign Data Wrapper for PostgreSQL linensed under
```

Comment:
Could you revert this file. It's our template.

Reply:
What exactly is not correct? This is not "The PostgreSQL Licence" or I have mistaken in `2024`? Does it means Toshiba Corporation doesn't upgrade this FDW in 2019, 2020, 2021, 2022 and 2023? Or any changes after 2018 doesn't protected by the license? I understand this is template, but what if this template is outdated? What do you law department think about this?

Reply:
Our rule does not have last modified year.     
And this FDW created based on mysql_fdw, so we keep the License same as mysql_fdw include the notice:
`Copyright (c) 2011 - 2016, EnterpriseDB Corporation`

And also, mysql_fdw does not have "The PostgreSQL Licence" notice.

Reply:
Thanks for explanation, reverted in https://github.com/pgspider/sqlite_fdw/pull/88/commits/6ff3790a8a7970f26cfb11e4d4472f24a678cb85

Reply:
Thank you for fixing. Please help me to resolve this code.

Reply:
I'll try to ask `mysql_fdw` authors about complete description of `mysql_fdw` license. (Note: `mysql_fdw` doesn't look like active project, still 19 PRs )